packages/headless/src/features/commerce/common/actions.ts

```diff
@@ -14,6 +17,7 @@ import {
   FacetSection,
   NumericFacetSection,
   ProductListingV2Section,
+  RecommendationV2Section,
```

Comment:
```suggestion
  RecommendationSection,
```

Reply:
We are not keeping sync with listings by dropping that